Automattic/wp-calypso
# Theme switching: remove Gutenberg prompt

#### Changes proposed in this Pull Request

Remove the prompt to opt-in to Gutenberg after switching themes. The editor preference that causes this prompt to appear (mainly for users who registered prior to 2018) is no longer used since Gutenberg became the default editor for all users last year.

Alternative to fix the issues described in D65357-code

#### Testing instructions

* Switch themes in Appearance > Themes
* You should _not_ see a message about activating Gutenberg, even if your uses account was registered prior to 2018, and the site does not have an editor preference set.

<img width="304" alt="image" src="https://user-images.githubusercontent.com/1699996/129090865-c8dff096-1a8e-4daf-85b1-cd40a2f56333.png">
